juju/juju
# Instancemutater

## Description of change

Add in the feature tag for instance-mutater.  If the feature flag is used, start the InstanceMutater worker.

## QA steps

1. `export JUJU_DEV_FEATURE_FLAGS=instance-mutater`
1. `juju bootstrap localhost --build-agent --config logging-config='<root>=DEBUG;unit=DEBUG;juju.worker.instancemutater=TRACE;juju.apiserver.instancemutater=TRACE'`
2. `juju debug-log -m controller`     <- verify that the instancemutater worker has started.
3. `juju debug-log -m controller --include-module juju.worker.instancemutater`   <-- verify message for watching machine when new machine created.

## Documentation changes

There should be no documentation changes, as this should be
the same feature but with a different implementation
